### PR TITLE
3755 export table

### PR DESCRIPTION
**Pull request by emitche**
Allow export of loans and organizations index to CSV

### CODE REVIEW

File: app/views/admin/loans/_grid_content.html.slim [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/ - g.blank_slate do
```
Comment (emitche) [inline app/views/admin/loans/_grid_content.html.slim]: Commenting this out is unnecessary.

File: app/views/admin/loans/_loans_grid.html.slim [hunk 2]
```diff
@@ -0,0 +1,52 @@
+= define_grid(@loans_grid, show_filters: :always,
+    html: {class: 'rowlink table-hover'}) do |g|
+
+    / = render "grid_content", g: g
+
+    / - g.blank_slate do
```
Comment (emitche) [inline app/views/admin/loans/_loans_grid.html.slim]: Same as above.

**Comment by emitche** [comment]
Visually, the index of organizations and loans may need some tweaks in terms of the filters and buttons. But CSV export functionally works in this feature.

File: app/controllers/admin/loans_controller.rb [hunk 3]
```diff
@@ -10,8 +10,18 @@ def index
       order: 'loans.signing_date',
       order_direction: 'desc',
       custom_order: { 'loans.signing_date' => 'loans.signing_date IS NULL, loans.signing_date' },
-      per_page: 50
+      per_page: 50,
+      name: 'loans',
+      enable_export_to_csv: true,
+      csv_file_name: 'loans'
```
Comment (hooverlunch) [inline app/controllers/admin/loans_controller.rb]: I don't think this is needed unless it differs from `name`?

File: app/controllers/admin/loans_controller.rb [hunk 4]
```diff
@@ -10,8 +10,18 @@ def index
...
     )
+
+    @csv_mode = true
+
+    export_grid_if_requested('loans' => 'loans_grid') do
```
Comment (hooverlunch) [inline app/controllers/admin/loans_controller.rb]: I don't think you need the argument here. I think loans -> loans_grid is the default. Let me know if I'm wrong. In general the Rails philosophy is convention over configuration so good to stick to that if possible.

**Comment by hooverlunch** [comment]
Looks like filter bar no longer has gray bg. Was this on purpose?

**Comment by hooverlunch** [comment]
Interesting issue when you try to open one of these in Excel:

image

It's because the first two chars of file are 'ID':

https://support.microsoft.com/en-us/kb/215591

**Comment by hooverlunch** [comment]
Great work!!!

**Comment by emitche** [comment]
It appears some stylesheets have been removed along the way. The index table's header with filters looks much better with a background.

**Pull request merged**